aepsych/strategy.py

```diff
@@ -307,6 +314,20 @@ def n_trials(self):
         return self.min_asks
 
     def add_data(self, x, y):
+        """
```

Comment:
As above, add the correct type hints here to ensure any static typing errors do show up.